app/src/main/java/com/android/unio/MainActivity.kt

```diff
@@ -42,10 +45,17 @@ class MainActivity : ComponentActivity() {
 fun UnioApp() {
   val navController = rememberNavController()
   val navigationActions = NavigationAction(navController)
+  val db = FirebaseFirestore.getInstance()
   val associationViewModel: AssociationViewModel = viewModel(factory = AssociationViewModel.Factory)
 
   val context = LocalContext.current
 
+  /*val imageRep = ImageRepositoryFirebaseStorage()
```

Comment:
What is this commented code ? Can we just remove it ?

Reply:
Oh yes I used it to upload an association image to the storage in order to check if it showed on the Explore screen, but I should've deleted it.